lib/lob/resources/us_zip_lookups.rb

```diff
@@ -0,0 +1,20 @@
+require "lob/resources/resource_base"
+
+module Lob
+  module Resources
+    class USZipLookups < Lob::Resources::ResourceBase
+
+      undef_method :list, :find, :create, :destroy
+
+      def initialize(config)
+        super(config)
+        @endpoint = "us_zip_lookups"
+      end
+
+      def lookup(options={})
+        submit :post, endpoint_url, options
+      end
```

Comment:
I think we'll probably want to keep the `create` method here, and the `lookup` method is duplicating that functionality. So we would only do

```php
undef_method :list, :find, :destroy
```

and we could remove this function.